spec/substitution_helper_spec.rb

```diff
@@ -92,7 +107,7 @@
           }
 
           it "unpublishes the existing published item" do
-            expect(existing_item.reload.state).to eq("unpublished")
+            expect(existing_item.reload.substitute?).to be_truthy
```

Comment:
since these actually return a boolean it's probably better to be specific with `be true`